examples/rvc-app/linux/main.cpp

```diff
@@ -15,25 +15,44 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+#include "RvcAppCommandDelegate.h"
 #include "rvc-device.h"
 #include <AppMain.h>
 
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
 
-RvcDevice * rvcDevice = nullptr;
+namespace {
+constexpr const char kChipEventFifoPathPrefix[] = "/tmp/chip_rvc_fifo_";
+NamedPipeCommands sChipNamedPipeCommands;
+RvcAppCommandDelegate sRvcAppCommandDelegate;
+} // namespace
+
+RvcDevice * gRvcDevice = nullptr;
 
 void ApplicationInit()
 {
-    rvcDevice = new RvcDevice(1);
-    rvcDevice->Init();
+    std::string path = kChipEventFifoPathPrefix + std::to_string(getpid());
+
+    if (sChipNamedPipeCommands.Start(path, &sRvcAppCommandDelegate) != CHIP_NO_ERROR)
+    {
+        ChipLogError(NotSpecified, "Failed to start CHIP NamedPipeCommands");
+        sChipNamedPipeCommands.Stop();
+    }
+
+    gRvcDevice = new RvcDevice(1);
```

Comment:
Please replace "1" with a constant (is this the endpoint number?)

Reply:
Is this necessary? The constructor is well documented and modern IDEs will anyway note what the input variable name is. It feels a bit much to be defining variables to be used only once.

Reply:
As a general principle the integer literals should be avoided, to improve the code readability. The GitHub web interface and a bunch of less sophisticated code editors (vi, Sublime etc) don't help the way Visual Code does :)

Reply:
Could at least `#define RVC_ENDPOINT 1` in main.cpp?

Reply:
See [here](https://github.com/project-chip/connectedhomeip/pull/29159/commits/2df7b61f90c8efa79269c3d1bdbc597da4dcf9ef).